src/classes/User.js

```diff
@@ -0,0 +1,66 @@
+import Ingredient from './Ingredient';
+
+class User {
+  constructor(usersData) {
+    this.name = usersData.name;
+    this.id = usersData.id;
+    this.pantry = usersData.pantry
+    this.favoriteRecipes = [];
+    this.weeklyFavorites = [];
+    this.filteredByName = [];
+  }
+
+  addFavorite(recipe) {
+    let favRecipe = this.favoriteRecipes;
```

Comment:
All of these `let` variables can be `const`. Using `Array.push` doesn't re-assing the variable, it just updates it, so `const` makes the most sense here.